src/web/js/repl-ui.js

```diff
@@ -85,7 +85,7 @@
 
     // the result of applying `displayResult` is a function that MUST
     // NOT BE CALLED ON THE PYRET STACK.
-    function displayResult(output, callingRuntime, resultRuntime, isMain) {
+    function displayResult(output, callingRuntime, resultRuntime, isMain, updateItems) {
```

Comment:
I know this is a bit hypocritical to ask here, given the lack of comments on the _other_ parameters, but could we get a brief description/comment on what `updateItems` is for?

Reply:
Will do. BTW, this PR will be gaining more updates. Do you want to wait to merge, or is it best to merge early (and often)?